poc/vdaf_pine_test.py

```diff
@@ -69,9 +70,9 @@ def test_result_share_length(self):
 class TestPineVdafEndToEnd(unittest.TestCase):
 
     def setUp(self):
-        self.l2_norm_bound = 1.0
-        self.dimension = 20
         self.num_frac_bits = 4
+        self.l2_norm_bound = math.floor(1.0 * 2**self.num_frac_bits)
```

Comment:
In other languages,  likely we write another constructor/initializer, instead of having callers to implement this. In Python, perhaps `@classmethod`?

Reply:
I'm not sure I understand. Do you mean computing `math.floor(1.0 * 2**self.num_frac_bits)`? 

The challenge is that, in taskprov, we will encode `l2_norm_bound` as a `uint64`, not a float. So this is what the constructor will get.

Reply:
That's a good idea, will do.